src/components/asset-list/RecyclerAssetList/index.tsx

```diff
@@ -877,8 +872,4 @@ export default connect(
     openSavings,
     openSmallBalances,
   })
-)(
-  withThemeContext(
-    React.memo(RecyclerAssetList, (_, curr) => curr.isBlockingUpdate)
```

Comment:
maybe this will be better to use
```
React.memo(RecyclerAssetList, (_, curr) => curr.isBlockingUpdate && !curr.isRefreshing
```

but IDK, if this works, let's do this. I'm just worried about the initial issue. Can you check this?

Anyway, I'm rewriting the list

Reply:
Skylar will check this out separately, esp since you'll be rewriting list anyway not so important to do now